theoplayer/how-to-guides/android/cmcd/getting-started.mdx

```diff
@@ -0,0 +1,33 @@
+# Getting started with CMCD on Android
+
+import Intro from '../../shared/cmcd/_getting-started-intro.mdx';
+
+<Intro />
+
+## Usage
+
+To enable CMCD, developers can add a CMCDTransmissionMode inside a SourceDescription.
+
+```kotlin
+TypedSource typedSource = new TypedSource.Builder("<HLS_STREAM_URL>")
+    .cmcdTransmissionMode(CMCDTransmissionMode.HTTP_HEADER)
+    .build();
+SourceDescription sourceDescription = new SourceDescription.Builder(typedSource)
+    .build();
+theoplayer.getPlayer().setSource(sourceDescription);
+```
+
+The preferred mode of transmission for HTTP requests is to use custom headers, but you can configure the transmission mode when creating the CMCD source. For example, to transmit via query arguments:
+
+```kotlin
+TypedSource typedSource = new TypedSource.Builder("<HLS_STREAM_URL>")
+    .cmcdTransmissionMode(CMCDTransmissionMode.QUERY_ARGUMENT)
```

Comment:
Pro-tip: you can make this change "pop out" a bit more with `// highlight-next-line` [(see docs)](https://docusaurus.io/docs/markdown-features/code-blocks#highlighting-with-comments).
```suggestion
    // highlight-next-line
    .cmcdTransmissionMode(CMCDTransmissionMode.QUERY_ARGUMENT)
```